tests/unit/amo/api/test_collections.js

```diff
@@ -90,13 +92,13 @@ describe(__filename, () => {
       }).toThrow('slug is required');
     });
 
-    it('throws an error when username is missing', () => {
+    it('throws an error when userId is missing', () => {
```

Comment:
I thought we were going to remove any of these tests where the code was switched to `invariant`.